src/include/daos_srv/vos.h

```diff
@@ -579,23 +588,27 @@ vos_obj_delete(daos_handle_t coh, daos_unit_oid_t oid);
  * \param oid	[IN]	Object ID
  * \param epoch	[IN]	Epoch for the fetch. It will be ignored if epoch range
  *			is provided by \a iods.
- * \param flags [IN]	conditional flags
+ * \param cond_flags [IN]
+ *			conditional flags
  * \param dkey	[IN]	Distribution key.
  * \param nr	[IN]	Number of I/O descriptors in \a ios.
  * \param iods	[IN/OUT]
  *			Array of I/O descriptors. The returned record
  *			sizes are also restored in this parameter.
- * \param size_fetch[IN]
- *			Fetch size only
+ * \param fetch_flags	[IN]
+ *			VOS fetch flags, VOS_FETCH_SIZE_ONLY or
+ *			VOS_FETCH_RECX_LIST.
+ * \param shadows [IN]	optional shadow recx/epoch lists, one for each iod.
```

Comment:
I think we need to add more meaningful comments for this, e.g. extents covered by these should not be returned by fetch function.

Reply:
ok, i can add some comments when refresh this patch or on following patch.